### PR TITLE
L1 documentation

### PR DESCRIPTION
**Pull request by brikendr**
- Camera info
- Camera upgrade
- Camera Detections
- Camera PTZ & Enabling/Disabling Autozoom

### CODE REVIEW

File: examples/huddly_l1/upgrade.js [hunk 1]
```diff
@@ -4,58 +4,55 @@ const { HUDDLY_L1_PID } = require('@huddly/sdk/lib/src/components/device/factory
...
 const sdk = new HuddlySdk(ipApi);
+let aceDevice;
+let isUpgrading = false;
+
 
 const applicationTeardown = () => {
     process.exit();
 }
 
+const performUpgrade = (attachedDevice, cpioFilePath) => {
```
Comment (eskilgh) [inline examples/huddly_l1/upgrade.js:16]: nice

File: examples/huddly_l1/detections.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+const ipApi = new HuddlyDeviceApiIp();
+const sdk = new HuddlySdk(ipApi);
+let manager;
+let detector;
+
+sdk.on('ATTACH', async (aceDevice) => {
+  if (aceDevice.pid == HUDDLY_L1_PID) {
+    manager = aceDevice;
+    detector = manager.getDetector({ objectFilter: ['person'], DOWS: true, rawDetections: true });
```
Comment (eskilgh) [inline examples/huddly_l1/detections.js:13]: remove rawDetections param

File: examples/huddly_l1/detections.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    detector.on('DETECTIONS', (data = []) => {
```
Comment (eskilgh) [inline examples/huddly_l1/detections.js:14]: I think the data = [] as default is misleading, as it should never really happen given proper implementation on the sdk.

**Review by eskilgh — CHANGES_REQUESTED** [review]

**Review by eskilgh — APPROVED** [review]
👍👍👍👍👍👍

**Review by pkhansen — APPROVED** [review]

**Pull request merged**